docs/source/getting_started/running_at_scale.ipynb

```diff
@@ -75,32 +85,67 @@
     "\n",
-    "### Pop-up\n",
-    "- Click DPS/MAS Operations menu -> Execute DPS Job\n",
-    "- Select your algorithm from the dropdown\n",
-    "- A new popup will ask for inputs; if it doesn’t take inputs, the popup will say so.\n",
-    "- Click OK again to view the ID for the job just submitted.\n",
+    "2. You can run your newly-registered Algorithm here. You will see it in the Submit Jobs tool in the Algorithm drop-down menu. If you open the dropdown menu, you can type a few letters to filter the list. Your Algorithm will be labeled with the name you put into the Algorithm Name field in the registration form you just submitted (in this example, `rob_test_registration_2406b` with version/branch `main`).\n",
     "\n",
-    "OR\n",
+    "![Jobs UI with new Algorithm](_static/register-7-jobsui.png)\n",
     "\n",
-    "### maap-py\n",
-    "Import the `maap-py` library: if in Jupyter, click the small blue MAAP button in the top left corner to automatically insert code. If using a script, add these lines manually at the top of your notebook: \n",
+    "3. Once you have selected the Algorithm that you just registered, the form will display the inputs. By default every job has a \"Job Tag\" field and a \"Resource\" dropdown. Additionally, any inputs that you configured in your algorithm during registration will be available; in this case we have the \"input_file\" field.\n",
+    "\n",
+    "For our test run, choose a Job Tag that's distinctive. In this example we used `test_run_2024b` but you should pick your own. For the Resource, we can over-ride the one that we used during Algorithm registration; for the sake of this example we chose the same one as before (maap-dps-work-8gb).\n",
+    "\n",
+    "The input_file can be any file that is publicly accessible to MAAP, for example any file on the web. As an example you can use:\n",
     "```\n",
-    "from maap.maap import MAAP\n",
-    "maap = MAAP()\n",
+    "https://photojournal.jpl.nasa.gov/jpeg/PIA17931.jpg\n",
     "```\n",
     "\n",
-    "Pass your algorithm’s name, version, required inputs, and username to the function `maap.submitJob (identifier is job- algo_name:algo_version)`\n",
-    "Check result: `maap.getJobResult()`\n"
+    "Leave the \"Publish to Content Metadata Repository\" section alone. This is an old placeholder feature that will be removed. You may not see that section of the Job submission form.\n",
+    "\n",
+    "![Filled Submit Job form](_static/run-2-filledform.png)\n",
+    "\n",
+    "4. Press **Submit Job**. A few seconds later a message should appear beneath the button indicating a successful job submission.\n",
+    "\n",
+    "![Submitted Job](_static/run-3-submitsuccess.png)\n",
+    "\n",
+    "5. Next, in the Launcher open the View Jobs tool (next to the Submit Jobs tool). If you do not see your test Job, you may need to refresh the table that opens up by pressing the button next to the \"Last updated\" message.\n",
```

Comment:
@rtapella Should we mention you can cancel jobs here?

Reply:
I think it is okay to just let them run for this guide. It should be a very small job.

Reply:
I meant so that we can mention it as a feature? In case they ever do large jobs down the line that they want to cancel?